api/model/contestant_long.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+type ContestantDetailed struct {
```

Comment:
Is it really necessary to separate this struct from the previous one?

If so, then, it would be better to keep them both in the same file, since they relate somehow